models/engine/db_storage.py

```diff
@@ -51,6 +51,20 @@ def all(self, cls=None):
                     new_dict[key] = obj
         return (new_dict)
 
+    def get(self, cls, id):
+        """ this method is to retrieve one object """
+        for value in self.all(cls).values():
+            if value.get('id') == id:
+                return value
+        return None
+
+    def count(self, cls=None):
+        """ this method is to count the number of object in storage """
+        if self.all():
+            return len(self.all())
```

Comment:
The class come for default then is None, no need to add a condition

Reply:
Hello, thanks for your observation, I considerated and maked the change